### PR TITLE
Sync clear history

### LINKED ISSUE #13004: Sync clear history on paired devices
As a user
When I clear history in a chat
I want that to be reflected on my paired devices

### Description

If you clear history on device A, it should be reflected on any paired device.

### PR DESCRIPTION
**Pull request by qfrank**
Closes #13004

### CODE REVIEW

File: scripts/update-status-go.sh
Comment (jakubgs) [inline scripts/update-status-go.sh]: No, do not do this. There's a reason it uses the `${X:=y}` format, because `y` is the default if `X` is not set:
```
STATUS_GO_OWNER=qfrank scripts/update-status-go.sh sync_clear_history
```
This is how it should be done.

**Review by jakubgs — CHANGES_REQUESTED** [review]

File: scripts/update-status-go.sh
Comment (qfrank) [reply scripts/update-status-go.sh]: ok, thank you @jakubgs

**Review by flexsurfer — APPROVED** [review]

**Comment by qoqobolo** [comment]
@qfrank thanks for your contribution!

### ISSUE 1: History is not cleared in a public chat on a secondary device if you join a public chat and clear its history (both actions) on a primary device while the secondary device is offline  

**Steps to reproduce:**
1. Recover the same account on device1 and device2
2. Sync device1 with device2
3. Close Status on device2
4. On device1 join a public chat, send messages
5. Clear history on device1
6. Open Status on device2 > log in > check the chat

**Expected behavior:** the public chat is added, no chat history
**Actual behavior:** the public chat is added, but chat history is not cleared

**Comment by qfrank** [comment]
> @qfrank thanks for your contribution!
> 
> ### ISSUE 1: History is not cleared in a public chat on a secondary device if you join a public chat and clear its history (both actions) on a primary device while the secondary device is offline
> **Steps to reproduce:**
> 
> 1. Recover the same account on device1 and device2
> 2. Sync device1 with device2
> 3. Close Status on device2
> 4. On device1 join a public chat, send messages
> 5. Clear history on device1
> 6. Open Status on device2 > log in > check the chat
> 
> **Expected behavior:** the public chat is added, no chat history **Actual behavior:** the public chat is added, but chat history is not cleared

Hi @qoqobolo , fix for issue 1 pushed, thanks for report :)

**Comment by qoqobolo** [comment]
Thanks @qfrank! Looks great to me :)
PR is tested and now awaiting approval from @jakubgs.
After that PR can be merged.

**Comment by flexsurfer** [comment]
thanks @qfrank so now we need to merge status-go with bumped version, and change version in this PR

File: status-go-version.json
Comment (jakubgs) [inline status-go-version.json]: No, this is wrong. You cannot use a fork in `status-go-version.json`, nor should you use a branch.

It should be a tagged release.

**Review by jakubgs — DISMISSED** [review]

File: status-go-version.json
Comment (flexsurfer) [reply status-go-version.json]: its just temporary for testing

Comment (jakubgs) [reply status-go-version.json]: Well, that's the only thing I saw wrong, since I can't review the clojure changes.

**Comment by qfrank** [comment]
> thanks @qfrank so now we need to merge status-go with bumped version, and change version in this PR

hi @flexsurfer , i just bumped version for status-go

**Comment by qfrank** [comment]
hi @flexsurfer , version in this PR changed :)

**Pull request merged**
